pkg/controller/seed-controller-manager/kubernetes/resources.go

```diff
@@ -673,6 +675,17 @@ func (r *Reconciler) ensureNetworkPolicies(ctx context.Context, c *kubermaticv1.
 		if err := reconciling.ReconcileNetworkPolicies(ctx, namedNetworkPolicyReconcilerFactorys, c.Status.NamespaceName, r.Client); err != nil {
 			return fmt.Errorf("failed to ensure Network Policies: %w", err)
 		}
+
+		// if the CiliumNetworkPolicy feature gate is active and CiliumNetworkPolicies exist on the Seed, create CiliumNetworkPolicies
```

Comment:
Nit:
```suggestion
		// if the CiliumNetworkPolicy feature gate is active and CiliumNetworkPolicies resource exist on the Seed, create CiliumNetworkPolicies
```

Reply:
Updated it a bit more than just the suggested change, PTAL.